README.md

```diff
@@ -1,26 +1,28 @@
-[![Build Status](https://travis-ci.com/dabapps/csv-wrangler.svg?token=apzD3FKHpTNKHAtAu9xC&branch=master)](https://travis-ci.com/dabapps/csv-wrangler)
 CSV Wrangler
 ===================
+[![Build Status](https://travis-ci.com/dabapps/csv-wrangler.svg?token=apzD3FKHpTNKHAtAu9xC&branch=master)](https://travis-ci.com/dabapps/csv-wrangler)
+[![pypi release](https://img.shields.io/pypi/v/csv-wrangler.svg)](https://pypi.python.org/pypi/csv-wrangler)
+
 Statically typed Python 3 CSV generation helpers.
 Write nicely typed CSV handling logic, with reorderable headers!
 
-Settings
---------
+## Getting Started
 
-Currently not on `pip`, as this isn't quite ready to hit the big time.  Whack this into your requirements file:
+### Installation
 
-```
-    git+git://github.com/dabapps/csv-wrangler.git@v0.1.3#egg=csv-wrangler
-```
+Install from PIP
```

Comment:
nitpick: You install _with_ `pip`. You install _from_ PyPI

Reply:
Very nitpick indeed, copied this from `django-db-queue`, but I do like correctness